### PR TITLE
doc-enhancement/update PATH variable

### PR DESCRIPTION
**Pull request by angad-singhh**
- This PR improves the documentation regarding update PATH variable section.
- The PR fixes the issue #7344

### CODE REVIEW

File: docs/cpp/config-mingw.md [hunk 1]
```diff
@@ -51,12 +51,13 @@ To install the MinGW-w64 toolchain, check out this video or follow the steps bel
...
 1. Enter `Y` when prompted whether to proceed with the installation.
 
-1. Add the path to your MinGW-w64 `bin` folder to the Windows `PATH` environment variable by using the following steps:
+1. Add the path of your MinGW-w64 `bin` folder to the Windows `PATH` environment variable by using the following steps:
    1. In the Windows search bar, type **Settings** to open your Windows Settings.
    1. Search for **Edit environment variables for your account**.
    1. In your **User variables**, select the `Path` variable and then select **Edit**.
    1. Select **New** and add the MinGW-w64 destination folder you recorded during the installation process to the list. If you used the default settings above, then this will be the path: `C:\msys64\ucrt64\bin`.
-   1. Select **OK** to save the updated PATH. You will need to reopen any console windows for the new PATH location to be available.
+   1. Once you have added the PATH in **Edit Environment Variable** window, select **OK**, and ensure to select **OK** again in the **Environment Variable** window to update the PATH.
```
Comment (ntrogh) [inline docs/cpp/config-mingw.md:59]:
```suggestion
   1. Select **OK**, and then select **OK** again in the **Environment Variables** window to update the `PATH` environment variable.
```

File: docs/cpp/config-mingw.md [hunk 2]
```diff
@@ -51,12 +51,13 @@ To install the MinGW-w64 toolchain, check out this video or follow the steps bel
...
+   1. You will have to open a NEW console window for the updated PATH to work.
```
Comment (ntrogh) [inline docs/cpp/config-mingw.md:60]:
```suggestion
      You have to reopen any console windows for the updated `PATH` environment variable to be available.
```

File: docs/languages/cpp.md [hunk 3]
```diff
@@ -92,12 +92,13 @@ To understand the process, let's install Mingw-w64 via [MSYS2](https://www.msys2
...
 1. Enter `Y` when prompted whether to proceed with the installation.
 
-1. Add the path to your MinGW-w64 `bin` folder to the Windows `PATH` environment variable by using the following steps:
+1. Add the path of your MinGW-w64 `bin` folder to the Windows `PATH` environment variable by using the following steps:
    1. In the Windows search bar, type **Settings** to open your Windows Settings.
    1. Search for **Edit environment variables for your account**.
    1. In your **User variables**, select the `Path` variable and then select **Edit**.
    1. Select **New** and add the MinGW-w64 destination folder you recorded during the installation process to the list. If you selected the default installation steps, the path is: `C:\msys64\ucrt64\bin`.
-   1. Select **OK** to save the updated PATH. For the new `PATH` to be available, reopen your console windows.
+   1. Once you have added the PATH in **Edit Environment Variable** window, select **OK**, and ensure to select **OK** again in the **Environment Variable** window to update the PATH.
```
Comment (ntrogh) [inline docs/languages/cpp.md:100]:
```suggestion
   1. Select **OK**, and then select **OK** again in the **Environment Variables** window to update the `PATH` environment variable.
```

File: docs/languages/cpp.md [hunk 4]
```diff
@@ -92,12 +92,13 @@ To understand the process, let's install Mingw-w64 via [MSYS2](https://www.msys2
...
+   1. You will have to open a NEW console window for the updated PATH to work.
```
Comment (ntrogh) [inline docs/languages/cpp.md:101]:
```suggestion
      You have to reopen any console windows for the updated `PATH` environment variable to be available.
```

**Review by ntrogh — CHANGES_REQUESTED** [review]
@angad-singhh thanks for creating this PR. I've updated the wording slightly. I've changed the "reopening of console windows" from a bulleted item to just a note.

**Comment by angad-singhh** [comment]
> @angad-singhh thanks for creating this PR. I've updated the wording slightly. I've changed the "reopening of console windows" from a bulleted item to just a note.

Thanks, for the suggestions. I have made commits for the same.

**Review by ntrogh — APPROVED** [review]
Thank you!

**Review by Tyriar — APPROVED** [review]

**Comment by angad-singhh** [comment]
> @angad-singhh please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.

@microsoft-github-policy-service agree

**Pull request merged**